src/main/ts/utils/Globals.ts

```diff
@@ -0,0 +1,50 @@
+import * as path from 'path';
```

Comment:
I was really happy to get this working, as it means it'll load the dom globals variable list based on the TS version you're using in the project. It works by parsing the `lib.dom.d.ts` file bundled in TypeScript and then converts it to an AST and extracts the variable names (it's based on how dom-globals generates its files).